crates/dojo/lang/src/attribute_macros/model.rs

```diff
@@ -41,18 +38,10 @@ impl DojoModel {
     /// * A RewriteNode containing the generated code.
     pub fn from_struct(db: &dyn SyntaxGroup, struct_ast: ItemStruct) -> PluginResult {
         let mut diagnostics = vec![];
-        let mut parameters = ModelParameters::default();
-
-        parameters.load_from_struct(
-            db,
-            &DOJO_MODEL_ATTR.to_string(),
-            struct_ast.clone(),
-            &mut diagnostics,
-        );
 
         let model_type = struct_ast.name(db).as_syntax_node().get_text(db).trim().to_string();
 
-        let model_version = parameters.version.to_string();
+        let model_type_snake = model_type.to_case(Case::Snake);
```

Comment:
We don't want to use snake case anymore, and seems not used in the patch? Same for events.